TestHost.entitlements

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+		<string>$(AppIdentifierPrefix)jp.co.rakuten.ios.sdk.deviceinformation</string>
```

Comment:
We should remove this from the repo.

Also, the keychain sharing group shouldn't have the AppIdentifierPrefix. See "Xcode implicitly adds an appId prefix to keychain sharing" on our tech debt record.

Reply:
The tests will fail without this but I suppose we can generate them somewhere else with plistbuddy

Reply:
I don't think the tests would fail if the keychain access group is injected into a `RDeviceIdentifier` initializer. The test setup could inject the same keychain access group that you put into the TestHost entitlements.

Reply:
Not sure if we're thinking the same thing but basically remove the entitlements and read the access-group from an external source:

1. put "access-group" value as a env var on CI
2. CI calls `plistbuddy` and writes "access-group" value to `Tests-Info.plist`
3. xcode tests scheme build phase uses `plistbuddy` to generate `TestHost.entitlements`
4. keychain takes a value as an intializer to use as its "access-group" while the unit tests should use the "access-group" value in its `Test-Info.plist`

Reply:
For now, it's fine to change this to "shared.keychain.access.group".

I don't want that sensitive information committed to master.